integration/eks-test/scripts/eks-DNS-test.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Testing service consumption with client-hello pod
+
+echo "verifying cross-cluster service consumption..."
+
+kubectl exec $CLIENT_POD -n $NAMESPACE /bin/sh -- curl --version &>/dev/null
```

Comment:
I think it actually makes more sense to use $KUBECTL_BIN, and point it at local kubectl in eks-common.sh, so if we change to a distributed version we only need to change the path in one place.

Reply:
Makes sense, will add change.